test/test_manifest.json

```diff
@@ -1246,6 +1246,14 @@
        "enableXfa": true,
        "type": "eq"
     },
+    {  "id": "xfa_issue13855",
+       "file": "pdfs/xfa_issue13855.pdf",
+       "md5": "0aaaf62ff979c7837c2c63df44456238",
+       "link": true,
+       "rounds": 1,
+       "enableXfa": true,
+       "type": "eq"
+    },
```

Comment:
Do you need to test all pages here, you would it suffice to limit it to only the affected page(s)?

(One way of keeping the overall runtime of the tests more manageable, is to not test an entire document unless absolutely necessary.)

Reply:
I had the same thought here and I said to myself that maybe it could useful to have all pages to catch future regressions.
In other words, could we consider that xfa stuff is in a good enough state to only need one page ?

Reply:
> In other words, could we consider that xfa stuff is in a good enough state to only need one page ?

I hope that the answer, at this point in time, would be *yes*.

Reply:
eh eh..., I really hope too and QA people said us that their xp with xfa was pretty good in general.